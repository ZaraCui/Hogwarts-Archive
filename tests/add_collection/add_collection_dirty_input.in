ADD COLLECTION data/books_dupes.csv
LIST TYPES
NUMBER COPIES
EXIT
