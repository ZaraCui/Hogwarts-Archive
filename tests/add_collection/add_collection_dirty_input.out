user: No such collection.

user: No spellbooks in system.

user: No spellbooks in system.

user: Ending Archive process.
